pandas/core/dtypes/cast.py

```diff
@@ -63,13 +64,18 @@
     ABCDataFrame,
     ABCDatetimeArray,
     ABCDatetimeIndex,
+    ABCExtensionArray,
     ABCPeriodArray,
     ABCPeriodIndex,
     ABCSeries,
 )
 from pandas.core.dtypes.inference import is_list_like
 from pandas.core.dtypes.missing import isna, notna
 
+if TYPE_CHECKING:
+    from pandas import Series
+    from pandas.api.extensions import ExtensionArray
```

Comment:
can you add a "# noqa: F401" here to make the linter happy

Reply:
Done - thanks!